SignalMessaging/categories/UIView+OWS.m

```diff
@@ -268,6 +268,8 @@ - (NSLayoutConstraint *)autoPinLeadingToSuperview
 
 - (NSLayoutConstraint *)autoPinLeadingToSuperviewWithMargin:(CGFloat)margin
 {
+    self.translatesAutoresizingMaskIntoConstraints = NO;
```

Comment:
We always want to disable `translatesAutoresizingMaskIntoConstraints` whenever we start using auto layout constraints to pin edges of the view.